src/components/Onboarding/Onboarding.tsx

```diff
@@ -36,12 +36,20 @@ const Onboarding: React.FC = () => {
                       />
                     </svg>
                   </div>
-                  <p className="mainTitle">Welcome to VIA!</p>
+                  <p className="mainTitle is-size-4">Welcome to VIA!</p>
 
                   <p className="mainBody">
                     VIA is a help to aid users with visual impairment. <br />
                     <br />
-                    Are you new here?
+                    In the photo analysis page, you can take a photo or upload
+                    one from your camera roll and the items detected in the
+                    image will be returned to you. You can also save these
+                    searches and edit the values found in the images.
+                    <br />
+                    <br />
```

Comment:
Why all the `<br />` tags.... feel like it would've been better to just use margins, but whatever. TIME